src/test/java/org/embulk/output/sftp/TestSftpFileOutputPlugin.java

```diff
@@ -396,121 +389,129 @@ public void testConfigValuesIncludingProxy()
     //     0 second
 
     @Test
-    public void testUserPasswordAndPutToUserDirectoryRoot()
+    public void testUserPasswordAndPutToUserDirectoryRoot() throws IOException
     {
         // setting embulk config
         final String pathPrefix = "/test/testUserPassword";
-        String configYaml = "" +
-                "type: sftp\n" +
-                "host: " + HOST + "\n" +
-                "port: " + PORT + "\n" +
-                "user: " + USERNAME + "\n" +
-                "password: " + PASSWORD + "\n" +
-                "path_prefix: " + pathPrefix + "\n" +
-                "file_ext: txt\n" +
-                "formatter:\n" +
-                "  type: csv\n" +
-                "  newline: CRLF\n" +
-                "  newline_in_field: LF\n" +
-                "  header_line: true\n" +
-                "  charset: UTF-8\n" +
-                "  quote_policy: NONE\n" +
-                "  quote: \"\\\"\"\n" +
-                "  escape: \"\\\\\"\n" +
-                "  null_string: \"\"\n" +
-                "  default_timezone: 'UTC'";
-
-        // runner.transaction -> ...
-        run(configYaml);
+
+        final ConfigSource formatterConfig = CONFIG_MAPPER_FACTORY.newConfigSource()
+                .set("type", "csv")
+                .set("newline", "CRLF")
+                .set("newline_in_field", "LF")
+                .set("header_line", "true")
+                .set("charset", "UTF-8")
+                .set("quote_policy", "NONE")
+                .set("quote", "\"")
+                .set("escape", "\\")
+                .set("null_string", "")
+                .set("default_timezone", "UTC");
+
+        final ConfigSource config = CONFIG_MAPPER_FACTORY.newConfigSource()
+                .set("type", "sftp")
+                .set("host", HOST)
+                .set("port", PORT)
+                .set("user", USERNAME)
+                .set("password", PASSWORD)
+                .set("path_prefix", pathPrefix)
+                .set("file_ext", "txt")
+                .set("formatter", formatterConfig);
+
+        this.embulk.runOutput(config, Paths.get(Resources.getResource("in.csv").getPath()));
 
         List<String> fileList = lsR(Lists.<String>newArrayList(), Paths.get(testFolder.getRoot().getAbsolutePath()));
-        assertThat(fileList, hasItem(containsString(pathPrefix + "001.00.txt")));
-        assertRecordsInFile(String.format("%s/%s001.00.txt",
+        assertThat(fileList, hasItem(containsString(pathPrefix + "000.00.txt")));
+        assertRecordsInFile(String.format("%s/%s000.00.txt",
```

Comment:
It was `001` because the `taskId` was set explicitly to `1` in `run()`. But, there is no such setting with `TestingEmbulk`, then it now starts from 0.